src/nexus/backend/tensorflow_model.h

```diff
@@ -35,6 +35,8 @@ class TensorflowModel : public ModelInstance {
 
   void Postprocess(std::shared_ptr<Task> task) final;
 
+  int64_t get_peak_bytes_in_use() final;
```

Comment:
`s/final/override/`

Let's leave the bad legacies there and have good practices for new code.

Reply:
Done.